kornia/geometry/epipolar/_metrics.py

```diff
@@ -111,3 +113,66 @@ def symmetrical_epipolar_distance(
     if squared:
         return out
     return (out + eps).sqrt()
+
+
+def left_to_right_epipolar_distance(pts1: Tensor, pts2: Tensor, Fm: Tensor) -> Tensor:
+    r"""Return one-sided epipolar distance for correspondences given the fundamental matrix.
+    This method measures the distance from points in the right images to the epilines of the
+    corresponding points in the left images as they reflect in the right images.
+
+    Args:
+       pts1: correspondences from the left images with shape
+         (*, N, 2 or 3). If they are not homogeneous, converted automatically.
+       pts2: correspondences from the right images with shape
+         (*, N, 2 or 3). If they are not homogeneous, converted automatically.
+       Fm: Fundamental matrices with shape :math:`(*, 3, 3)`. Called Fm to
+         avoid ambiguity with torch.nn.functional.
+
+    Returns:
+        the computed Symmetrical distance with shape :math:`(*, N)`.
+    """
+    KORNIA_CHECK_IS_TENSOR(pts1)
+    KORNIA_CHECK_IS_TENSOR(pts2)
+    KORNIA_CHECK_IS_TENSOR(Fm)
+
+    if (len(Fm.shape) < 3) or not Fm.shape[-2:] == (3, 3):
```

Comment:
you might want to implement `KORNIA_CHECK_SHAPE`
https://github.com/kornia/kornia/blob/master/kornia/testing/__init__.py#L186